nextflow_schema.json

```diff
@@ -270,6 +284,13 @@
                     "description": "Specify the number of high-quality bases in the library to be retained",
                     "fa_icon": "fas fa-bullseye",
                     "help_text": "Removes the worst reads until only the specified value of bases remain, useful for very large read sets. If the input read set is less than the specified value, this setting will have no effect. _Modified from [Filtlong documentation](https://github.com/rrwick/Filtlong)_\n\n> Modifies tool parameter(s):\n> - Filtlong: `--keep_percent`"
+                },
+                "longread_qc_qualityfilter_minquality": {
```

Comment:
Given this only applies to nanoq, maybe we should indicate this... I guess we could change the name of the parmaeter but this would require a major bump e.g. 1.1 -> 2 (rather than 1.2)

Alternatively for the help descriptions we prepend for this and the other tools in this section if it's a tool-specific parameter

So here we could have maybe: `Nanoq only: specify minimum average read quality filter`, and above, `longread_qc_qualityfilter_keeppercent` could be `Filtlong only: specify the percent of high-quality bases to be retained`?

Reply:
Updated!